libsolidity/ast/AST.h

```diff
@@ -657,6 +657,24 @@ class VariableDeclaration: public Declaration
 public:
 	enum Location { Default, Storage, Memory, CallData };
 
+	static std::string locationToString(VariableDeclaration::Location _location)
+	{
+		switch (_location)
+		{
+			case Location::Memory:
+				return "memory";
+			case Location::Storage:
+				return "storage";
+			case Location::CallData:
+				return "calldata";
+			case Location::Default:
+				return "none";
+			default:
```

Comment:
Please remove the default case so that we get a compiler error here if we change the enum at some point.